core/trino-main/src/test/java/io/trino/execution/resourcegroups/TestResourceGroups.java

```diff
@@ -57,7 +58,8 @@
 
 public class TestResourceGroups
 {
-    @Test(timeOut = 10_000)
+    @Test
+    @Timeout(10)
```

Comment:
You can move the default to the class-level and this will be a default for all of the methods.

Reply:
Not all tests have a timeout. I prefer to keep the specifications as tight as possible.

Reply:
Sure